File: Codigo de los 4 lenguajes/C#/EjemploModificadores.cs
using System;

class ModificadoresDemo
{
    public string Publico = "soyPublico";
    private string Privado = "soyPrivado";
    protected string Protegido = "soyProtegido";
    string PorDefecto = "soyPorDefecto";
    public static string Estatico = "soyEstatico";

    public void MostrarTodos()
    {
        Console.WriteLine("publico: " + Publico);
        Console.WriteLine("privado: " + Privado);
        Console.WriteLine("protegido: " + Protegido);
        Console.WriteLine("porDefecto: " + PorDefecto);
        Console.WriteLine("estatico: " + Estatico);
    }
}

public class EjemploModificadores
{
    public void Mostrar()
    {
        Console.WriteLine(" Ejemplo: Modificadores de acceso");
        ModificadoresDemo d = new ModificadoresDemo();
        d.MostrarTodos();

        Console.WriteLine("Acceso estático directo: " + ModificadoresDemo.Estatico);
        Console.WriteLine();
    }
}

File: Codigo de los 4 lenguajes/js/EjemploModificadores.js
// EjemploModificadores.js
export default class EjemploModificadores {
  mostrar() {
    console.log("=== Ejemplo: Modificadores de acceso ===");

    class ModificadoresDemo {
      constructor() {
        this.publico = "soyPublico";
        this._privado = "soyPrivado"; 
        this.protegido = "soyProtegido";
        this.porDefecto = "soyPorDefecto";
      }
      mostrarTodos() {
        console.log("publico:", this.publico);
        console.log("privado:", this._privado);
        console.log("protegido:", this.protegido);
        console.log("porDefecto:", this.porDefecto);
        console.log("estatico:", ModificadoresDemo.estatico);
      }
      static estatico = "soyEstatico";
    }

    const d = new ModificadoresDemo();
    d.mostrarTodos();
    console.log("Acceso estático directo:", ModificadoresDemo.estatico);
    console.log();
  }
}
